src/rprblender/engine/engine.py

```diff
@@ -79,10 +79,17 @@ def depsgraph_instances(self, depsgraph: bpy.types.Depsgraph):
                 yield instance
 
     def cache_blur_data(self, depsgraph: bpy.types.Depsgraph):
-        cur_frame = depsgraph.scene.frame_current
+        scene = depsgraph.scene
+        cur_frame = scene.frame_current
+
+        subframe = {  # following Cycles behavior
+            'START': 1.0,  # shutter opens at the current frame, means at the end
+            'CENTER': 0.5,
+            'END': 0.0,  # shutter closes at the current frame, means at the start
```

Comment:
So if the shutter is closing at the current frame shouldn't we blur from 
n-1 -> N
and start would be 
N-> N+1
and center would be N -.5 -> N + .5

Reply:
Yes, that looks closer to Cycles. Done.